Images.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Images</title>
+    <link rel="stylesheet" href="images.css" />
+    <link rel="stylesheet" href="component.css">
+  </head>
+  <body>
+    <section class="header"></section>
+    <section>
+      <div class="sidebar">
+        <a href="index.html">Home</a>
+        <a href="Alert.html">Alert</a>
+        <a href="Avatar.html">Avatar</a>
+        <a href="badges.html">Badges</a>
+        <a href="Buttons.html">Buttons</a>
+        <a href="cards.html">Cards</a>
+        <a href="Images.html" class="active">Images</a>
+        <a href="Input.html">Input</a>
+      </div>
+
+      <div class="content">
+        <main class="text-center">
+          <h3>Cards</h3>
+        </main>
+        <h3>Responsive Image</h3>
+        <div class="img-content">
+            <div>
+              <img class="img-responsive" src="Images/iphoneimg.png" alt="" srcset="">
```

Comment:
Provide the `alt` tag in all the images. This will improve the accessibility

Reply:
I Have updated